lib/core/presentation/pages/collaborator/collaborator_discover/collaborator_discover_page.dart

```diff
@@ -38,7 +39,19 @@ class _CollaboratorDiscoverPageState extends State<CollaboratorDiscoverPage> {
           Padding(
             padding: EdgeInsets.only(right: Spacing.medium),
             child: InkWell(
-              onTap: () {},
+              onTap: () {
+                showCupertinoModalBottomSheet(
+                  context: context,
+                  backgroundColor: colorScheme.surface,
+                  topRadius: Radius.circular(30.r),
+                  builder: (mContext) {
+                    return const FractionallySizedBox(
```

Comment:
why need this ? if you want bottomsheet to expand full height, use expand = true property in showCupertinoModalBottomSheet